### PR TITLE
Fix signatures not being stored as text on syncs

### PR DESCRIPTION
**Pull request by gerrod3**
fixes: #1245

### CODE REVIEW

File: CHANGES/1245.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+``CollectionVersionSignature`` data is now stored as plain text instead of bytes.
```
Comment (mdellweg) [inline CHANGES/1245.misc:1]: Isn't this a bugfix?

Do you think we need a data-repair command?

Comment (newswangerd) [reply CHANGES/1245.misc:1]: It hasn't shipped with anything yet, so this should be fine.

**Review by newswangerd — APPROVED** [review]

File: CHANGES/1245.misc (hunk 1 above)
Comment (mdellweg) [reply CHANGES/1245.misc:1]: So do we need to backport any of this?

Comment (mdellweg) [reply CHANGES/1245.misc:1]: I believe not. This is really a followup to https://github.com/pulp/pulp_ansible/pull/845 .

**Pull request merged**